src/components/atoms/sidebar-item.tsx

```diff
@@ -0,0 +1,42 @@
+import { useHash } from 'react-use';
+
+import type { Section } from '@/types/data';
+import type { IconName } from '@/types/icon';
+
+import Icon from './icon';
+
+export interface Props {
+  section: Section;
+  icon: IconName;
+}
+
+const MAIN_SECTION: Section = 'main';
```

Comment:
Może wyrzućmy to do pliku ze stałymi, jakby miało się w przyszłości przydać do innych funkcjonalności